### ISSUE TITLE
Wierd new bug (breaks the entire program)

### ISSUE THREAD

**Issue by Zayoe4** [TheFuckingRat/RatPoison#305]
Stuck here when trying to load the program.

![image](https://user-images.githubusercontent.com/10657721/74292912-dfe5b980-4cfe-11ea-94b6-89d5e7e5940e.png)

**Comment by TheFuckingRat** [comment]
Make sure you fully close everything before reopening. If you've touched MENU_APP make sure it's correct. Fully restart your computer if you need to. I see a white square in the cmd as if you've clicked and dragged into the cmd, this pauses it's actions (typically), click on the cmd and hit enter and see if that fixes it.

**Comment by Zayoe4** [comment]
I'll give that a try

**Comment by Zayoe4** [comment]
Yah it worked after a restart. Thanks!!

**Comment by 10urshin** [comment]
> Stuck here when trying to load the program.
> 
> ![image](https://user-images.githubusercontent.com/10657721/74292912-dfe5b980-4cfe-11ea-94b6-89d5e7e5940e.png)

In selection mode, cmd doesn't process the stdout of the running program, which most of times caused the running program to hang. 

TLDR; when in selection mode cmd pauses the cheat. Make sure its not in selection mode.
